frontend/App.tsx

```
@@ -2,8 +2,8 @@ import { useState, useEffect } from "react";
 import "./App.css";
 
 import { googleLogout, useGoogleLogin } from "@react-oauth/google";
-import axios from "axios";
 import MenuBar from "./MenuBar";
+import { auth } from "./auth";
```

ESLint

[error] 'auth' is defined but never used.

(@typescript-eslint/no-unused-vars)